### PR TITLE
Add first version of u-track downloads page

### PR DESCRIPTION
**Pull request by sbesson**
Similar to #58, this PR provides the framwork for generating a downloads page for the U-Track partner project. The corresponding jobs should be http://ci.openmicroscopy.org/view/Consortium/job/U-TRACK-release/? and http://ci.openmicroscopy.org/view/Consortium/job/U-TRACK-release-downloads.

/cc @hflynn

### CODE REVIEW

**Comment by sbesson** [comment]
See http://downloads.openmicroscopy.org/u-track/2.1.1-rc1/.

**Comment by hflynn** [comment]
Looks good in general. Not sure of the value of the section links at the top when the page is so short though.

File: utrack_downloads.html [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+<div class="entry-content">
+
+<h2>u-track @VERSION@ Downloads</h2>
+
+<p>
+  <strong><a href="#code">Code</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="#legacy">Previous versions</a> </strong></p>
+
+<ul>
+<li>
+<p>The software is currently distributed as a source code bundle and requires MATLAB to be installed with some toolboxes described in the documentation.</a>.</p>
```
Comment (hflynn) [inline utrack_downloads.html]: You have an extra '.' at the end of this sentence

**Comment by sbesson** [comment]
Should be built on http://downloads.openmicroscopy.org/u-track/2.1.1-rc1/ with some of the comments by @hflynn addressed.
Also added a quickstart line post discussion with @pwalczysko.
We may want to use another icon than the OME source code. Any shiny proposition, @gusferguson?

**Comment by hflynn** [comment]
Looks much better, the text at the top is better too.
Do you want to add a link to http://www.openmicroscopy.org/site/support/omero5/developers/Matlab.html to the line about installing the OMERO MATLAB plugin? (or downloads.openmicroscopy.org/omero to remove the version number issue?)

**Comment by hflynn** [comment]
Lovely, thank you!

**Comment by gusferguson** [comment]
@sbesson - without a logo for u-track - I could just put in text - otherwise I can do you a temporary logo using the name

**Comment by sbesson** [comment]
@gusferguson: no logo yet indeed. Thoughts about using the MATLAB logo and the software name (like for OMERO.matlab)?

**Comment by sbesson** [comment]
Merging post u-track 2.1.1 release. Logo will be updated in following versions.

**Comment by sbesson** [comment]
--rebased-to #63

**Pull request merged**